models/cow_protocol/arbitrum/cow_protocol_arbitrum_trades.sql

```diff
@@ -0,0 +1,225 @@
+{{  config(
+        
```

Comment:
with the above comment on schemas, plz add here instead

Reply:
Makes sense, adjusted